### PR TITLE
Replace entityId by recordId in the front end.

### PR DESCRIPTION
**Pull request by ehconitin**
Hey @lucasbordeau, I replaced every `entityId` with `recordId` in the frontend.

### Some clarifications:

1. At [this line](https://github.com/ehconitin/twenty/commit/0b207d26b6e78da349af34cf44585d89cb59adb7#diff-b4f415dd1f060307ad61f64394ee96b2912f6015e26c7fd2eab4b8c6a84d2d07L14), I changed `recordId` to `selectedRecordId` because that component has `entityId` defined in code at [this line](https://github.com/ehconitin/twenty/commit/0b207d26b6e78da349af34cf44585d89cb59adb7#diff-b4f415dd1f060307ad61f64394ee96b2912f6015e26c7fd2eab4b8c6a84d2d07L55) which was to be changed to `recordId`. To avoid repeated constants, I changed the arguments to `selectedRecordIds`.

2. At the following links:
   - [File 1](https://github.com/ehconitin/twenty/commit/0b207d26b6e78da349af34cf44585d89cb59adb7#diff-52b780bdd4cfc582ca7e1b457dbbd63733bfbb7790fc421054bbd2dbf0389e16)
   - [File 2](https://github.com/ehconitin/twenty/commit/0b207d26b6e78da349af34cf44585d89cb59adb7#diff-6d47cb9a59dfcf6b1495937084ae799a61da6afccb21208f04ce8e1f5afca0e4)
   - [File 3](https://github.com/ehconitin/twenty/commit/0b207d26b6e78da349af34cf44585d89cb59adb7#diff-821815783f9968f1da3cd437fc9d1d1666d12dc331d279cc5fbd9817bc2df2bf)

   It seems to be the tests. As I can see, it is checking for both `objectFilterDropdownSelectedEntityIdState` and `objectFilterDropdownSelectedRecordIdsState`. Since `entityIds` are supposed to be removed and `recordedId` state is already being checked, I commented out all the `entityidState` code. If they are to be removed or uncommented, please let me know, and I will do as expected.

### CODE REVIEW

File: packages/twenty-front/src/modules/object-record/object-filter-dropdown/hooks/__tests__/useFilterDropdown.test.tsx [hunk 1]
```diff
@@ -178,35 +178,35 @@ describe('useFilterDropdown', () => {
...
-      result.current.setObjectFilterDropdownSelectedEntityId(mockResult);
-    });
-
-    await waitFor(() => {
-      expect(result.current.objectFilterDropdownSelectedEntityId).toBe(
-        mockResult,
-      );
-    });
-  });
+  // it('should set objectFilterDropdownSelectedEntityId', async () => {
```
Comment (lucasbordeau) [inline packages/twenty-front/src/modules/object-record/object-filter-dropdown/hooks/__tests__/useFilterDropdown.test.tsx:181]: Ok we should get rid of `objectFilterDropdownSelectedEntityId` state, but we should rewrite this test for `objectFilterDropdownSelectedRecordIdsState`

**Review by lucasbordeau — COMMENTED** [review]
- Could you create a util `getRecordFieldInputId` that returns `${recordId}-${fieldDefinition?.metadata?.fieldName}` from params.
- Could you clean `recordId: recordId` => `recordId,`

**Review by lucasbordeau — COMMENTED** [review]
Nice job !

**Comment by ehconitin** [comment]
- Could you create a util `getRecordFieldInputId` that returns `${recordId}-${fieldDefinition?.metadata?.fieldName}` from params.
> Yes, Ill work on it.

- Could you clean recordId: recordId => recordId,
> I am sorry but could u please give an example on where this should be cleaned up.

**Comment by lucasbordeau** [comment]
> I am sorry but could u please give an example on where this should be cleaned up.

In `usePersistField` 

```tsx
recordStoreFamilySelector({ recordId: recordId, fieldName }),
```

To : 

```tsx
recordStoreFamilySelector({ recordId, fieldName }),
```

**Comment by ehconitin** [comment]
Got it, my bad should have cleaned in the first place. But will do and let you know.

**Comment by ehconitin** [comment]
@lucasbordeau 

- Could you create a util getRecordFieldInputId that returns ${recordId}-${fieldDefinition?.metadata?.fieldName} from params. - **Done**
- Could you clean recordId: recordId => recordId. - **Done**

Could u please review and let me know if there are more changes to be made.

Thanks :)

File: packages/twenty-front/src/modules/object-record/object-filter-dropdown/hooks/__tests__/useFilterDropdown.test.tsx (hunk 1 above)
Comment (ehconitin) [reply packages/twenty-front/src/modules/object-record/object-filter-dropdown/hooks/__tests__/useFilterDropdown.test.tsx:181]: So what should I in this case, keep test cases as it is and create a new issue to rewrite the tests?
What do u suggest?

**Review by FelixMalfait — APPROVED** [review]
@lucasbordeau  is off this week so I will merge like this to avoid further conflicts. Great work thanks again!

**Pull request merged**